lib/adskHydraSceneBrowser/lib/CMakeLists.txt

```diff
@@ -136,10 +136,6 @@ target_include_directories(${TARGET_NAME}
 # -----------------------------------------------------------------------------
 # compiler configuration
 # -----------------------------------------------------------------------------
-# QT_NO_KEYWORDS prevents Qt from defining the foreach, signals, slots and emit macros.
-# this avoids overlap between Qt macros and boost, and enforces using Q_ macros.
-set_target_properties(Qt6::Core PROPERTIES INTERFACE_COMPILE_DEFINITIONS QT_NO_KEYWORDS)
```

Comment:
I actually don't agree with this change. I think that only the libraries in maya-hydra that need Qt should have the qt target property and the qt linked libs.

Reply:
I like sharing Qt-related CMake code, but I agree with @seando-adsk that Qt dependencies should be reserved to those targets that need it.

Reply:
Is that not the case? This line only runs if Qt was found (it is inside an `if (Qt6_FOUND)`), and from my understanding this is setting a property on the `Qt6::Core` target, not maya-hydra targets. I don't think this links Qt to maya-hydra targets either, does it?

Reply:
Yes it is setting a property on the Qt6::Core library. But here it would only do it for this library. By moving it to the make CMakeLists.txt you are setting that property for every library in MayaHydra. My feeling is that the individual libraries (in MayaHydra) that use Qt should each decide whether or not they want this property set.

Reply:
Ah that makes sense yeah, I've reverted it to be on a per-target basis, thanks